src/components/common/AddNameTranslation.vue

```diff
@@ -78,3 +78,10 @@ export default class AddNameTranslation extends Vue {
   protected removeTranslation (): void {}
 }
 </script>
+
+<style lang="scss" scoped>
+// un-bold the text field label
+.v-text-field :deep(.v-label) {
+  font-weight: normal;
+}
```

Comment:
Before:
![image](https://user-images.githubusercontent.com/10002889/218227971-7abc4a96-b732-41a1-8db0-65a71b3b516b.png)
and
![image](https://user-images.githubusercontent.com/10002889/218227982-f17a1289-d35c-45f3-8cbf-406374a77d9c.png)

After:
![image](https://user-images.githubusercontent.com/10002889/218228003-5360354c-a24d-4c49-9095-5b9f0620fb37.png)
and
![image](https://user-images.githubusercontent.com/10002889/218228011-2c38c99a-3b99-49b1-a64d-1f02b705077c.png)

This is now consistent with all other textfields in our UIs.